### ISSUE TITLE
TQAddJobEx check tq_length length potential problem

### ISSUE THREAD

**Issue by jzy-chitong56** [SMUnlimited/AMAI#484]
If this JOB is for `RESET GUARDPOSITION`, it should not be overlooked

**Comment by SMUnlimited** [comment]
Yes there is a max length limit same with any array/group/hashtable, in this case if thats hit jobs are ignored, should be unlikely if we only pass the current ai's units and not all units of other players into jobs.

**Comment by jzy-chitong56** [comment]
Consider immediately restoring the alert point

Simultaneous excretion unit group

**Comment by jzy-chitong56** [comment]
https://github.com/jzy-chitong56/AMAI-CN/commit/36a833de89eea1c0aae15e76bf8a21c4b84883a1
